drivers/ethernet/eth_mcux.c

```diff
@@ -347,7 +347,7 @@ static int eth_tx(struct net_if *iface, struct net_pkt *pkt)
 	irq_unlock(imask);
 
 	if (status) {
-		SYS_LOG_ERR("ENET_SendFrame error: %d\n", status);
+		SYS_LOG_ERR("ENET_SendFrame error: %d\n", (int)status);
```

Comment:
Could you remove the extra \n which causes empty line to be printed

Reply:
Done consistently for all messages in the file.